sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -466,6 +466,12 @@ object DateTimeUtils {
     }
   }
 
+  def stringToDateAnsi(s: UTF8String, zoneId: ZoneId): Int = {
+    stringToDate(s, zoneId).getOrElse {
```

Comment:
can you also update `stringToTimestampAnsi` to follow this code style?

Reply:
sure, updated.